addons/event/models/event_event.py

```diff
@@ -176,6 +176,20 @@ def _default_event_mail_ids(self):
         'Start sale date', compute='_compute_start_sale_date',
         help='If ticketing is used, contains the earliest starting sale date of tickets.')
 
+    def name_get(self):
+        if not self.env.context.get('registration'):
+            return super().name_get()
+        res = []
+        for event in self:
+            name = (
+                _("%s (%s seats remaining)",
```

Comment:
Better use a form like ``_(%(event_name) (%(seats_count)s seats remaining)``, event_name=event.name, seats_count=event.seats_available`` . This eases translations as it gives context for people that translate the whole string without having the code around.